packages/hebao_v1/contracts/base/BaseWallet.sol

```diff
@@ -36,14 +37,15 @@ contract BaseWallet is Wallet, AddressSet, ReentrancyGuard
     address internal _owner;
 
     bytes32 internal constant MODULE = keccak256("__MODULE__");
+    bytes32 internal constant SUB_ACCOUNTS = keccak256("__SUB_ACCOUNT__");
```

Comment:
Do we need to keep track of modules that are sub accounts in the wallet? Seems like this is mainly useful off-chain where we can just loop over all modules and not something we really need to store in the wallet on-chain.